### DISCUSSION TITLE
HDR contect playback on Nvidia Shield

### DISCUSSION THREAD

**Discussion by grannymaster** [jellyfin/jellyfin#8651] [General]
Hey, like many other people I'm moving from PLEX to Jellyfin and so far I've been loving it.

I'm having trouble playing HDR content on my Nvidia Shield. I honestly don't know why is not working, so I can't really provide any information about what I've done, as I am lost. :(

Some info:
- Jellyfin is running as LXC container on Proxmox with Intel Quick Sync passthough. iGPU is detected by the CT.
- File that I've been testing with "verygoodmovie.INTERNAL.HDR.2160p.WEB.H265-DEFLATE.mkv", PLEX had no problems playing this in HDR.
- Hardware acceleration and encoding are turned on. Here is all my playback settings : 
https://i.imgur.com/AJGqtCr.png
https://i.imgur.com/iqng1yI.png
https://i.imgur.com/ScEAWFX.png
- Jellyfin version 10.8.5

Do I need to change some settings, or is this a compatibility issue or something else?

**Comment by Tokynet** [comment]
@grannymaster Did you make any progress on figuring this out? I'm on having the same issues. MediaFile plays properly on Plex app but its purple/green'ish on Jellyfin.

I have Jellyfin running as a container in UnRaid with Quick-Sync enabled.
Jellyfin app and Shield are up-to-date as of today.

**Reply by grannymaster** [reply]
I kinda fixed it. I just re-installed jellyfin and that somehow magically fixed it.
